assets/sass/_langchoose.scss

```diff
@@ -117,6 +117,13 @@ $tab-font-size: 14px !default;
     cursor: pointer;
 }
 
+.langtab-tabs {
```

Comment:
Instead of putting this in `.mdl-tabs` I kept it more specific to the lang chooser, as I plan to extract out the tabs into their own more general purpose component, because I want to reuse tabs for things other than the language chooser. For example, one piece of feedback we got on the new get started guide (from Twitter) was to just have tabs for platform selecting an OS when installing Pulumi (as it wasn't immediately obvious we supported Windows since we do platform detection and only show a single platform, macOS, when viewing from an iPhone).